lib/rules/index.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+module.exports = {
```

Comment:
Uh oh, is this a merge conflict?

Reply:
yupp, Got little confusion but resolved now